charts/opentelemetry-collector/templates/daemonset.yaml

```diff
@@ -37,6 +37,11 @@ spec:
       {{- include "opentelemetry-collector.pod" ($podData | mustMergeOverwrite (deepCopy .)) | nindent 6 }}
       hostNetwork: {{ .Values.hostNetwork }}
       {{- with .Values.dnsPolicy }}
-      dnsPolicy: {{ . }}
+      dnsPolicy: 
+        {{ toYaml . | nindent 4 }}
```

Comment:
dnsPolicy templating was correct and we don't need to change it.

Reply:
My bad - Reverting this